.lintstagedrc

```diff
@@ -2,16 +2,13 @@
   "*.{js,ts,tsx}": [
     "eslint",
     "jest --config=utils/test/jest.config.js --findRelatedTests",
-    "prettier --write",
-    "git add"
```

Comment:
We're getting husky warnings locally. Looks like they've added this functionality in by default.